rfcs/moderation-log.md

```diff
@@ -0,0 +1,124 @@
+# RFC - Moderation Action Log
+
+Ever since the [Ayo.js Code of
+Conduct](https://github.com/ayojs/ayo/blob/latest/CODE_OF_CONDUCT.md) was
+ratified, the Moderation Team has been entrusted with following the processes
+described there in order to promote the general health and safety of the
+community.
```

Comment:
🎨 [copy editing/style] The introduction reflects temporal closeness of the CoC ratification and Moderation Team introduction. It is more relevant today, since these things are all new.

Reading this sentence in a year or two, the aspect of when the CoC was ratified is less relevant. As a matter style only, I suggest a more timeless rephrase along the lines of:

```
The Moderation team is entrusted with following the processes described in the [Ayo.js Code of
 +Conduct](https://github.com/ayojs/ayo/blob/latest/CODE_OF_CONDUCT.md) in order to promote the general health and safety of the
 +community.
```

Reply:
After playing around with it and trying out your rewording, I think it's best to just remove this introduction altogether. A better, more relevant introduction already exists not much further below, in the `Keeping a Log` section.